src/ipfs/publish_to_ipfs.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
-
 source /app/.env
-/usr/local/bin/ipfs name publish `/usr/local/bin/ipfs add -Q -r /app/repository`
+export PATH=~$PATH:/usr/local/bin:/usr/local/openjdk-11/bin:/usr/bin
+export JAVA_HOME=/usr/local/openjdk-11/
+
+hash=$(ipfs add -Q -r /app/repository)
+echo $hash
+curl -X POST "https://ipfs.komputing.org/api/v0/pin/add?arg=$hash&recursive=true"
+ipfs name publish $hash
+
+/app/source_verify_ens_updater/bin/source_verify_ens_updater $IPFS_SECRET /app/repository
```

Comment:
the ENS updater should get no IPFS secret

Reply:
Removed.